test/listing.py

```diff
@@ -66,20 +66,6 @@ def test_list_mbeds_ext_mandatory_fields_exist(self):
             self.assertIn('platform_name_unique', mbed)
             self.assertIn('daplink_version', mbed)
 
-    def test_list_platforms(self):
-        platforms = self.mbeds.list_platforms()
-        self.assertIs(type(platforms), list)
-        for p in platforms:
-            self.assertIs(type(p), str)
-
-    def test_list_platforms_ext(self):
-        platforms = self.mbeds.list_platforms_ext()
-        self.assertIs(type(platforms), dict)
-        for p in platforms:
-            self.assertIs(type(p), str)
-            self.assertIs(type(platforms[p]), int)
-            self.assertTrue(platforms[p] >= 0)
```

Comment:
Even if functions are deprecated, should we still test them? Seems like when they're removed it'd be appropriate to stop testing them.

If you feel strongly the other way then no worries, just my initial reaction.

Reply:
I guess. I don't feel strongly either way.